src/foam/net/web/HTTPRequest.js

```diff
@@ -92,6 +95,16 @@ foam.CLASS({
     {
       class: 'Boolean',
       name: 'cache'
+    },
+    {
+      class: 'Int',
+      name: 'timeout',
+      value: 3000
```

Comment:
This is way too low, also browsers automatically timeout network calls so this might be redundant